test_utils/fake_catalog_api.py

```diff
@@ -566,6 +570,174 @@
     }
 }
 
+FAKE_CATALOG_COURSE_PAGINATED_RESPONSE = {
+    'count': 3,
+    'next': 'http://testserver/api/v1/catalogs/1/courses?page=3',
+    'previous': 'http://testserver/api/v1/catalogs/1/courses?page=1',
+    'results': [
+        {
+            'owners': [
+                {
+                    'description': None,
+                    'tags': [],
+                    'name': '',
+                    'homepage_url': None,
+                    'key': 'edX',
+                    'certificate_logo_image_url': None,
+                    'marketing_url': None,
+                    'logo_image_url': None,
+                    'uuid': FAKE_UUIDS[1]
+                }
+            ],
+            'uuid': FAKE_UUIDS[2],
+            'title': 'edX Demonstration Course',
+            'prerequisites': [],
+            'image': None,
+            'expected_learning_items': [],
+            'sponsors': [],
+            'modified': '2017-03-03T07:34:19.322916Z',
+            'full_description': None,
+            'subjects': [],
+            'video': None,
+            'key': 'edX+DemoX',
+            'short_description': None,
+            'marketing_url': None,
+            'level_type': None,
+            'course_runs': []
+        }
+    ]
+}
+
+FAKE_SEARCH_ALL_COURSE_RESULT = {
+    "title": "edX Demonstration Course",
+    "min_effort": None,
+    "marketing_url": "course/course-v1:edX+DemoX+Demo_Course/about",
+    "image_url": "https://business.sandbox.edx.org/asset-v1:edX+DemoX+Demo_Course+type"
+                 "@asset+block@images_course_image.jpg",
```

Comment:
I copied this straight from discovery-business.sandbox.edx.org/api-docs -- I didn't think it was harmful, as it's just a fake and temporary resource on the business sandbox itself. No bad-intentioned person would look at this and say, "found the weak spot".